src/QuartzHostedService/QuartzHostedService.csproj

```diff
@@ -1,19 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
Sorry, but i can't approve this. TargetFramework shold be netstandard (not netcoreapp).
Why do you want to change TargetFramework?

Reply:
Because IHostBuilder 
 and other interfaces only in netcoreapp 3.0  “ public static IHostBuilder ConfigureQuartzHost(this IHostBuilder builder)” 
https://github.com/mukmyash/QuartzHostedService/blob/c7bed530f90b3cfc6082c9c24a6c414be54f58e4/src/QuartzHostedService/IServiceCollectionExtensions.cs#L40

Reply:
In the future, only .Net 5  ，So no more netstandard 2.0 is needed.

Reply:
`IHostBuilder` included in _Microsoft.Extensions.Hosting.Abstractions_ package, whose TargetFramework set netstandard2.0. Maybe use this package instead _Microsoft.AspNetCore.App_?

https://www.nuget.org/packages/Microsoft.Extensions.Hosting.Abstractions/